apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -63,6 +67,12 @@ export const styles = {
     lineHeight: '52px',
     width: 210
   },
+  col5: {
```

Comment:
It might be better to name the column for what it contains than for its number.  Easier to understand, and easier to rearrange.